### PR TITLE
AC-429: Can login without specifying location for systems that have login locations

### PR DESCRIPTION
**Pull request by KredkaOfficial**
### Description of what I changed
I've added checking value of session location Spinner on login page. If nothing is selected then login button is inactive.

### Issue I worked on
https://issues.openmrs.org/browse/AC-429

### CODE REVIEW

**Comment by dkayiwa** [comment]
@f4ww4z do you wanna take a look at this? 😊

**Comment by KredkaOfficial** [comment]
I made changes to this project, I guess it is what I was supposed to do

File: openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.java [hunk 1]
```diff
@@ -142,6 +142,14 @@ private boolean validateNotEmpty(EditText editText) {
         return StringUtils.notEmpty(editText.getText().toString());
     }
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.java]: Can you add a Java doc comment that describes this method? E.g.
```suggestion
/**
   * <Method description here>
   * @return True if a location is selected or no location needed for the OpenMRS instance used.
   */
```

**Comment by f4ww4z** [comment]
@KredkaOfficial Remember to hit the 'request code review' button in the JIRA page as mentioned at https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Pull request merged**